_ml-commons-plugin/api/model-apis/update-model.md

```diff
@@ -130,3 +130,53 @@ PUT /_plugins/_ml/models/9uGdCJABjaMXYrp14YRj
 }
 ```
 
+#### Example request: Updating the model interface 
+
+You can update a model's interface to define input and output schemas. This is useful for models that lack a default interface or require customization.  
```

Comment:
Suggest user read more details from https://opensearch.org/docs/latest/ml-commons-plugin/api/model-apis/register-model/#the-interface-parameter

Reply:
addressed in here addressed in https://github.com/opensearch-project/documentation-website/pull/9246/commits/f291cf082d3a97df3ba906a5ede89ace131da54d